docs/pvacseq/output_files.rst

```diff
@@ -172,80 +190,144 @@ all_epitopes.tsv and filtered.tsv Report Columns
+   * - ``ic50_MT``
+     - Median ic50 binding affinity of the best-binding mutant epitope across all prediction algorithms used
+   * - ``ic50_WT``
+     - Median ic50 binding affinity of the corresponding wildtype epitope across all prediction algorithms used.
+   * - ``percentile_MT``
+     - Median ic50 binding affinity percentile rank of the best-binding mutant epitope across all prediction algorithms used
+   * - ``percentile_WT``
+     - Median ic50 binding affinity percentile rank of the corresponding wildtype epitope across all prediction algorithms used.
+   * - ``RNA_expr``
+     - Gene expression value for the annotated gene containing the variant.
+   * - ``RNA_VAF``
+     - Tumor RNA variant allele frequency (VAF) at this position.
+   * - ``RNA_Depth``
+     - Tumor RNA depth at this position.
+   * - ``DNA_VAF``
+     - Tumor DNA variant allele frequency (VAF) at this position.
+   * - ``tier``
+     - A tier corresponding to the suitability of variants for use in vaccines.
 
-The pVACseq Neoeptiope Priority Rank
-____________________________________
+The pVACseq Aggregate Report Tiers
+__________________________________
 
-Each of the following 4 criteria are assigned a rank-ordered value (worst = 1):
+To bin a variant in a tier, the best binding epitope is evaluated as follows:
 
-- B = Rank of the mutant IC50 binding affinity, with the lowest being the best.
-  If the ``--top-score-metric`` is set to ``median`` (default) the ``Median MT Score``
-  is used. If it is set to ``lowest`` the ``Best MT Score`` is used.
-- F = Rank of ``Fold Change`` between MT and WT alleles, with the highest being the best.
-- M = Rank of mutant allele expression, calculated as (``Gene Expression`` * ``Tumor RNA VAF``), with the highest being the best.
-- D = Rank of ``Tumor DNA VAF``, with the highest being the best.
+.. list-table::
+   :header-rows: 1
 
-A score is calculated from the above ranks with the following formula: ``B + F + (M * 2) + (D / 2)``. This score is then converted to a rank (best = 1).
+   * - Tier
+     - Citeria
+   * - ``Pass``
+     - Median MT Score < 500 and (Tumor RNA VAF * Gene Expression) > 3 and
+       Tumor DNA VAF > (clonal VAF / 2) and not an anchor residue
+   * - ``Relaxed``
+     - Median MT Score < 1000 and (Tumor RNA VAF * Gene Expression) > 1 and
+       Tumor DNA VAF > (clonal VAF / 2) and not an anchor residue
+   * - ``Anchor``
+     - Median MT Score < 1000 and (Tumor RNA VAF * Gene Expression) > 1 and
+       Tumor DNA VAF > (clonal VAF / 2) and an anchor residue
+   * - ``Subclonal``
+     - Median MT Score < 1000 and (Tumor RNA VAF * Gene Expression) > 1 and
+       Tumor DNA VAF < (clonal VAF / 2) and not an anchor residue
+   * - ``LowExpr``
+     - Median MT Score < 1000 and low expression and
+       Tumor DNA VAF > (clonal VAF / 2) and not an anchor residue
+   * - ``NoExpr``
+     - Gene Expression == 0 and Tumor DNA VAF == 0 and expression is not low
```

Comment:
A little confused on what "expression is not low" means (particularly with gene expression = 0)?